sentry_sdk/tracing.py

```diff
@@ -1,23 +1,20 @@
 import uuid
 import random
-
-from datetime import datetime, timedelta
+from datetime import datetime, timedelta, timezone
 
 import sentry_sdk
 from sentry_sdk.consts import INSTRUMENTER
 from sentry_sdk.utils import is_valid_sample_rate, logger, nanosecond_time
-from sentry_sdk._compat import datetime_utcnow, utc_from_timestamp, PY2
 from sentry_sdk.consts import SPANDATA
 from sentry_sdk._types import TYPE_CHECKING
 
 
 if TYPE_CHECKING:
-    import typing
-
     from typing import Any
     from typing import Dict
     from typing import Iterator
     from typing import List
+    from typing import Mapping
```

Comment:
```suggestion
    from collections.abc import Mapping
```

`typing.Mapping` is a [deprecated alias](https://docs.python.org/3/library/typing.html#typing.Mapping) to `collections.abc.Mapping`